### PR TITLE
Support new versions of vep cache

### PR DESCRIPTION
**Pull request by apregier**
Since the release of GRCh38, we need to specify which reference version we want when downloading the vep cache.

### CODE REVIEW

File: lib/perl/Genome/Db/Ensembl/Api.pm [hunk 1]
```diff
@@ -27,19 +27,21 @@ sub create {
     my @package_names = $self->package_names;
     #OLD CVS location
     #my $base_url = "'http://cvs.sanger.ac.uk/cgi-bin/viewvc.cgi/PACKAGENAME.tar.gz?root=ensembl&only_with_tag=branch-ensembl-VERSION&view=tar'";
-    my $base_url = "https://github.com/Ensembl/PACKAGENAME/archive/release/VERSION.zip";
+    #OLD github location
+    #my $base_url = "https://github.com/Ensembl/PACKAGENAME/archive/release/VERSION.zip";
```
Comment (tmooney) [inline lib/perl/Genome/Db/Ensembl/Api.pm]: Do we need to keep these old locations around?

**Comment by tmooney** [comment]
+1 :construction:

**Comment by acoffman** [comment]
+1 Looks good to me!

File: lib/perl/Genome/Db/Ensembl/Api.pm (hunk 1 above)
Comment (apregier) [inline lib/perl/Genome/Db/Ensembl/Api.pm]: Good point - looks like they now have git branches for all legacy releases, so I removed this old code.  (It's always there in the history anyway!)

**Pull request merged**